lib/breakers/dashboard.rb

```diff
@@ -0,0 +1,53 @@
+require 'sinatra/base'
+require 'json'
+
+require 'byebug'
+
+module Breakers
+  class Dashboard < Sinatra::Base
```

Comment:
So I think your idea of having a separate gem or building the UI into prometheus or grafana might just be preferred, because otherwise you have to introduce sinatra as a dependency and more than likely this dashboard interface is not something you would ever want running across load-balanced app servers -- instead it would probably run on its own stand-alone app server.

I think this is fine for now, it would just be nice to not have Sinatra as an additional dependency in vets-api, and it's a concern that I think others (open source community) would share if they were using Grape, Cuba, Hanami, Rails, etc and want to use this gem.